src/impl/configurator_from_yaml.cpp

```diff
@@ -409,6 +410,25 @@ namespace soralog {
       }
     }
 
+    auto at_fault_node = sink_node["at_fault"];
+    if (at_fault_node.IsDefined()) {
+      if (not at_fault_node.IsScalar()) {
+        errors_ << "W: Property 'at_fault' of sink node is not scalar\n";
```

Comment:
I think it would be helpful to also write what it is.

Reply:
Explanation provided [here](https://github.com/qdrvm/soralog/blob/5769bdcf66bace4678f7f5cd95ac799afcdd456c/example/01-simple/logger.yml#L36). And sure it is debt to make documentation, I know...

Reply:
I think it would be pretty convenient if the error message contained a short description of what at_fault is supposed to be. This way there's no need to go search somewhere in soralog how to fix it when it appears.

Reply:
Error report now will contain link to doc